server/game/cards/01-DTR/MakeTheSmartChoice.js

```diff
@@ -0,0 +1,49 @@
+const ActionCard = require('../../actioncard.js');
+
+class MakeTheSmartChoice extends ActionCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Make the Smart Choice',
+            playType: ['shootout'],
+            target: {
+                activePromptTitle: 'Choose a dude',
+                choosingPlayer: 'current',
+                cardCondition: { location: 'play area', controller: 'any', participating: true },
+                cardType: ['dude']
+            },
+            handler: context => {
+                // "Reduce a dude's bullets by their influence."
+                const influence = context.target.influence;
+                if(influence) {
+                    this.applyAbilityEffect(context.ability, ability => ({
+                        match: context.target,
+                        effect: ability.effects.modifyBullets(-1 * influence)
+                    }));
+                    this.game.addMessage(
+                        '{0} uses {1} to reduce {2}\'s bullets by their influence ({3})',
+                        context.target.controller,
+                        this,
+                        context.target,
+                        influence
+                    );
+                }
+                // "The dude's controller may move them home booted."
+                context.game.promptForYesNo(context.target.controller, {
+                    title: 'Send home booted?',
+                    onYes: () => {
+                        this.game.shootout.sendHome(context.target, context)
+                            .thenExecute(() => this.game.addMessage(
+                                '{0} uses {1} to send {2} home booted', 
```

Comment:
I would like to use the `X uses Y` if X is the one triggering the ability, while in this case he is forced to use it.
My suggestion would be something like 
`X sends Y home because he Z`
or
`X decides to send Y home because of the Z`

with _Z_ being the Make the Smart Choice card. Or some combination.

Reply:
I see what you're getting at, but that makes for improper English. My suggestion here is to template the output message as `X makes the smart choice to send Y home booted.`

Reply:
![image](https://user-images.githubusercontent.com/1410427/118382885-7cf06b80-b5ae-11eb-9271-d72cbe07dc3f.png)

Reply:
I see where you're going with the `because of Z`, but that's a bit awkward in English.
Suggesting we stick with `X  uses Y to send Z home` message template here.

![image](https://user-images.githubusercontent.com/1410427/118383036-5bdc4a80-b5af-11eb-8308-61698169b7d7.png)

Reply:
another option could be `X makes the smart choice and sends Y home.` that would be grammatically correct English and natural language.

Reply:
final idea on this: `X decides to Y and send Z home`, for example: `Player1 decides to Make the Smart Choice and sends Bobo home`.

Reply:
👍 That is basically what I had in mind, but obviously mi english is not that great. And I realize that many `traitReaction` abilities are not worded correctly because I used the same "style".